llvm/docs/tutorial/MyFirstLanguageFrontend/LangImpl05.rst

```diff
@@ -447,8 +447,7 @@ from a starting value, while the condition ("i < n" in this case) is
 true, incrementing by an optional step value ("1.0" in this case). If
 the step value is omitted, it defaults to 1.0. While the loop is true,
 it executes its body expression. Because we don't have anything better
-to return, we'll just define the loop as always returning 0.0. In the
-future when we have mutable variables, it will get more useful.
```

Comment:
I removed the final sentence of this paragraph because I realized that the tutorial never actually does anything with the for-loop return value in future chapters.